### ISSUE TITLE
hololens build

### ISSUE THREAD

**Issue by peter-clemenko** [felixse/FluentTerminal#898]
a hololens build would be very useful for using the terminal to connect to remote devices from hololens

**Comment by felixse** [comment]
I don't think this will be possible due to the separate full trust process we need, but if someone wants to try it go ahead
